internal/cluster/cluster_test.go

```diff
@@ -37,6 +38,12 @@ func getDefaultConfig() Config {
 	return cfg
 }
 
+func getOcsConfig() ocs.OCSValidator {
```

Comment:
You will not need it if you work with interfaces and mocks, this is a unit-test adding something like this will break the concept of `unit`